GOVERNANCE.md

```diff
@@ -79,9 +79,9 @@ a standard TC motion.
 Changes to TC membership should be posted in the agenda, and may be
 suggested as any other agenda item (see "TC Meetings" below).
 
-No more than 1/3 of the TC members may be affiliated with the same
+No more than 1/4 of the TC members may be affiliated with the same
```

Comment:
That adds an upper limit on the ratio, but if the TC is going to grow over time it may be wise to also add an upper limit on the actual number of individuals affiliated with the same employer that can be on the TC. Sort of depends on how large the TC is expected to get though.